libs/core/mappers/broadcasted-time.mapper.ts

```diff
@@ -0,0 +1,27 @@
+import { BroadCastedTimeDto } from '../dtos/broadcasted-time.dto';
+import { BroadCastedTime } from '../models/broadcasted-time.model';
+
+export namespace broadCastedTimeMapper {
```

Comment:
Namespace should follow PascalCase convention:
```suggestion
export namespace BroadCastedTimeMapper {
```